lib/tools.js

```diff
@@ -8,51 +8,49 @@ const chalk = require('chalk')
 const warning = chalk.keyword('yellow')
 const info = chalk.keyword('green')
 
-let packages
-
-try {
-  const output = execSync('npx lerna ls --all --json')
-  packages = JSON.parse(output.toString()).map(
-    packageItem => `../packages/${packageItem.name}`
-  )
-} catch (error) {
-  console.log(warning('No packages found'))
-  process.exit(0)
+const pkgDirs = getPkgDirs()
+function getPkgDirs() {
+  try {
+    const packages = execSync('npx lerna ls --all --json')
+    return JSON.parse(packages.toString()).map(
+      ({ location }) => location
+    )
+  } catch (error) {
+    console.log(warning('No packages found'))
+    process.exit(0)
+  }
 }
 
 function cleanEnv() {
-  for (const pathDir of packages) {
-    const env = path.resolve(__dirname, `${pathDir}/.env`)
+  for (const pkgDir of pkgDirs) {
+    const env = `${pkgDir}/.env`
```

Comment:
shouldn't this take into account the current directory as it did before?

Reply:
No, `package.location` is an absolute path from root.